lib/components/viewers/RouteRow.js

```diff
@@ -68,6 +106,31 @@ const SimpleRouteRenderer = styled.span`
   padding: 0;
 `
 
+const PatternButton = styled.button`
+  background: transparent;
+  border: none;
+  border-radius: 50%;
+  color: #004f96;
+  opacity: ${(props) => (props.display ? 100 : 0)};
+  height: 40px;
+  margin-right: 8px;
+  position: relative;
+  transition: all ease-in-out 0.2s;
+  width: 40px;
+  z-index: ${(props) => (props.display ? 2 : -2)};
+
+  svg {
+    margin-top: -3px;
+    height: 22px !important;
+    width: 22px !important;
```

Comment:
this is a massive nitpick, but can we set the width and height on the container instead? I'm getting a weird non-circle highlight
<img width="94" alt="Screenshot 2023-03-29 at 10 56 59 AM" src="https://user-images.githubusercontent.com/86619099/228580169-0b5a6daa-be03-4b27-90a8-437759c1e691.png">

Reply:
Ooh good catch. So moving the height and width out of the svg makes it so small, but I fixed this by adding a `min-width` to the button! Let me know if you're still seeing it.